src/arrays.js

```diff
@@ -1,61 +1,69 @@
-const getNthElement = (index, array) => {
-  // your code here
-};
+const getNthElement = (index, array) => array[index % array.length];
 
-const arrayToCSVString = array => {
-  // your code here
-};
+const arrayToCSVString = array => array.toString();
 
-const csvStringToArray = string => {
-  // your code here
-};
+const csvStringToArray = string => string.split(',');
 
 const addToArray = (element, array) => {
-  // your code here
+  array.push(element);
 };
 
-const addToArray2 = (element, array) => {
-  // your code here
-};
+const addToArray2 = (element, array) => array.concat(element);
 
-const removeNthElement = (index, array) => {
-  // your code here
-};
+const removeNthElement = (index, array) => array.splice(index, 1);
 
 const numbersToStrings = numbers => {
-  // your code here
+  return numbers.map(element => element.toString());
 };
 
 const uppercaseWordsInArray = strings => {
-  // your code here
+  return strings.map(letter => letter.toUpperCase());
 };
 
 const reverseWordsInArray = strings => {
-  // your code here
+  return strings.map(word =>
+    word
+      .split('')
+      .reverse()
+      .join('')
+  );
```

Comment:
good, but I would change `word` to `string`. By convention, you should try to give the name of the parameter in your callback function the singular version of the name of the array, if possible. As the name of the array is `strings` you should use the singular `string`